code/TemplateStudioForUWP/Templates/_comp/MT/Project._VB/.template.config/template.json

```diff
@@ -7,16 +7,16 @@
   "name": "MVVMToolkit.Project.VB",
   "identity": "MVVMToolkit.Project.VB",
   "shortName": "MVVMToolkit.Project.VB",
-  "tags": {
-    "language": "VisualBasic",
-    "type": "item",
-    "ts.type": "composition",
-    "ts.platform" : "Uwp",
-    "ts.version": "1.0.0",
-    "ts.compositionOrder": "0",
-    "ts.compositionFilter": "$frontendframework == MVVMToolkit & ts.type == project",
-    "ts.licenses": "[Microsoft.Toolkit.MVVM](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/main/License.md)"
-  },
+    "tags": {
```

Comment:
Remove excess whitespace here.

Reply:
The whitespace here is automatically indented by the editor because the indent size of the project is 4
https://github.com/microsoft/TemplateStudio/blob/61f4f9f2d8eccd1f25455417468671a1e9b72a5c/.editorconfig#L12

Reply:
I recently started formatting the json files in the WinUI extension to standardize on the 4 space tabs configured in editorconfig. If you format the json files you changed so the whole file uses 4 spaces, then we can resolve these comments.

Reply:
I've solved it.

Reply:
Thanks.